test/test-initialization-errors.js

```diff
@@ -92,10 +92,10 @@ it('errors are thrown in connectedCallback when template result fails to render'
         strings: {},
       };
     }
-    static template(html, { repeat }) {
+    static template(html, { keyedMap }) {
```

Comment:
The required changes to the current test suite were _super minimal_. Really, it's only due to slight differences in how keyed lists are managed.

This doesn't mean that this isn't a large breaking change — if anything it just reflects the fact that the behavior of x-element should really be decoupled from the underlying templating engine implementation.